dist/images/.gitignore

```diff
@@ -1,5 +1,6 @@
 ovn-k8s-cni-overlay
 ovn-kube-util
 ovnkube
+ovndbchecker
```

Comment:
we can change it when we add the bash db management logic?

Reply:
I think it makes more sense to start with a more generic name. It isn't a huge refactor right? If it is then we can defer it.

Reply:
it will need changes to all the files and the binary everywhere. i would rather get this PR merged and then do it in a separate PR when we move over the bash code to go.

Reply:
ok